react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```

Comment:
Although it works without it, it is best to set the `href` equal to `"#"` for navigating to the "home" route. It is better practice and shows intention not that you might have accidentally left the `href` blank. 

```JAVASCRIPT
<a href="#" className="navbar-brand">
```